script/build-module.js

```diff
@@ -45,18 +50,23 @@ async function parseFile (file) {
   // derive category from file path
   // {locale}/docs/api/{filename} -> api
   file.category = file.relativePath
-    .split(path.sep)
+    .split('/') // path.sep => /, separator in file.relativePath is just / in any OS
     .slice(2, -1)
-    .join(path.sep)
+    .join('/')
 
   // nice categories for use in nav
   file.categoryFancy = categoryNames[file.category]
 
-  file.href = path.join('/docs', file.category, file.slug)
+  file.href = `/docs/${file.category}/${file.slug}`.replace('//', '/')
```

Comment:
Why is this needed? I think `path.join` will take care of duplicate slashes.

Reply:
I think `.replace` is simple and just work on both of Windows, Unix, and Linux env.
On Windows, `path.join` will return a path has separators are `\` in it. (must be replaced `\` to `/`, It takes up to 2 steps while we just need to replace it once for simplicity).

Reply:
Ok 👍